docs/gaia/gaiacli.md

```diff
@@ -183,10 +183,40 @@ gaiacli tx sign --validate-signatures signedSendTx.json
 
 You can broadcast the signed transaction to a node by providing the JSON file to the following command:
 
-```
+```bash
 gaiacli tx broadcast --node=<node> signedSendTx.json
 ```
 
+### Query Transactions
+
+#### Matching a set of tags
+
+You can use the transaction search command to query for transactions that match a specific set of `tags`, which are added on every transaction.
+
+```bash
+gaiacli query txs --tags=<tag1>:<value1>&<tag2>:<value2>
+```
+
+::: tip Note
+
+You can find a list of available `tags` on each of the SDK modules:
+
+- [Common tags](https://github.com/cosmos/cosmos-sdk/blob/d1e76221d8e28824bb4791cb4ad8662d2ae9051e/types/tags.go#L57-L63)
```

Comment:
I don't think linking to old revisions is a great idea, this will quickly get out of date.

Instead can we put canonical tags in specific files (`tags.go`) in each module?

Reply:
yes, I added this note in the meantime. See #2913